src/core/client/auth/components/ForgotPassword.css

```diff
@@ -0,0 +1,9 @@
+.root {
+  width: 330px;
+  padding-top: calc(5 * var(--spacing-unit));
+}
```

Comment:
This styling should be applied in App.css` (in the `App.ts` file the root style needs to be used in a div container, I forgot that)

Reply:
👍